test/parallel/test-buffer-copy.js

```diff
@@ -144,3 +155,10 @@ assert.strictEqual(b.copy(c, 512, 0, 10), 0);
     assert.strictEqual(c[i], e[i]);
   }
 }
+
+// https://github.com/nodejs/node/issues/23668: Do not crash for invalid input.
+b.copy(c, 'not a valid offset');
```

Comment:
Nit: An assertion after an invalid copy which ensures that nothing actually copied would be good. If the behaviour changes later on, this assertion would be able to catch that.

Reply:
@thefourtheye I’ve documented the behaviour – it is is not copying anything, it is coercing,  values to integer, i.e. `'not a valid offset'` becomes `0` (as in, `('foo' | 0) === 0`)